backend/clubs/views.py

```diff
@@ -205,13 +205,19 @@ def parse_year(field, value, operation, queryset):
             return {}
 
         def parse_int(field, value, operation, queryset):
+            if operation == "in":
+                values = value.strip().split(",")
+                sizes = [int(size) for size in values if size]
+                return {f"{field}__in": sizes}
+
```

Comment:
My solution to the same problem that you solve below, instead I added a "in" operation. We can go with either solution if you wanna take mine out no worries.

Reply:
The default operation is actually "in" if you don't specify "and", see line 214/219 below.

I like how you made it explicit though! Definitely makes it less confusing for someone reading the code. One option could be to add a comment above line 214/219 saying that the default behavior is to do "in".